contrib/packs/actions/pack_mgmt/setup_virtualenv.py

```diff
@@ -132,8 +134,15 @@ def _create_virtualenv(self, virtualenv_path):
         self.logger.debug('Creating virtualenv in "%s" using Python binary "%s"' %
                           (virtualenv_path, python_binary))
 
-        cmd = ['virtualenv', '-p', python_binary, '--system-site-packages', virtualenv_path]
-        exit_code, _, stderr = run_command(cmd=cmd)
+        cmd = [virtualenv_binary, '-p', python_binary]
+        cmd.extend(virtualenv_opts)
+        cmd.extend([virtualenv_path])
+        self.logger.debug('Running command "%s" to create virtualenv.', ' '.join(cmd))
+
+        try:
+            exit_code, _, stderr = run_command(cmd=cmd)
+        except OSError:
+            raise Exception('Virtualenv binary "%s" doesn\'t exist.' % virtualenv_binary)
```

Comment:
The original error message was "No such file or directory" without naming the file. This was confusing. Thus this new message.